README.md

```diff
@@ -9,194 +9,192 @@
+- Email [help@finos.org](mailto:help@finos.org) to be added to the meeting invite directly, or find the meeting in the [FINOS Community Calendar](https://calendar.google.com/calendar/embed?src=finos.org_fac8mo1rfc6ehscg0d80fi8jig%40group.calendar.google.com).
 
 ### Need help?
-Email [fdc3@finos.org](mailto:fdc3@finos.org) if you need help getting started in the FDC3 Community. If you encounter technical difficulties accessing repositories, joining Slack, mailing lists or meetings, please email [help@finos.org](mailto:help@finos.org).
+
+- Email [fdc3@finos.org](mailto:fdc3@finos.org) if you need help getting started in the FDC3 Community. 
+- If you encounter technical difficulties accessing repositories, joining Slack, mailing lists or meetings, please email [help@finos.org](mailto:help@finos.org).
+
+## Roadmap
+
+Work on FDC3 is split into several "streams".  You can view details of the streams on our [Roadmap page](https://fdc3.finos.org/fdc3-roadmap).  All standardization work is carried out in this GitHub repo.
+
+## Contributing
+
+If you'd like to contribute to the FDC3 standard, or have noticed something that needs correcting, the first step is to [raise an issue on the FDC3 Github Repo](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) and describe what you'd like to see changed. There are a number of issue templates that you can choose from [here](https://github.com/finos/FDC3/issues/new/choose).
+
+
+
+_NOTE:_
+
+- Issues that change the Standard usually need discussion. You can post comments directly on the issue or can ask for it to be added to a Standards Working Group meeting agenda by emailing [fdc3@finos.org](mailto:fdc3@finos.org), sending a message to the [#fdc3 channel on the FINOS slack](https://finos-lf.slack.com/messages/fdc3/) or tag the FDC3 maintainers (`@finos/fdc3-maintainers`) in your issue.
+- To implement changes in the FDC3 repository resolving an issue please read our [contribution guidelines](CONTRIBUTING.md). 
+- Contributions merged into the master branch of the FDC3 repository will form part of the next pre-draft of the FDC3 Standard (as defined by the [FDC3 Governance document](./GOVERNANCE.md)), which must be approved by a Standard Working Group vote before it is accepted as a draft and subsequently released as the next version of the Standard.
+- Commits and pull requests to FINOS repositories will only be accepted from those contributors with an active, executed Individual Contributor License Agreement (ICLA) with FINOS, _OR_ who are covered under an existing and active Corporate Contribution License Agreement (CCLA) executed with FINOS. Commits from individuals not covered under an ICLA or CCLA will be flagged and blocked by the [Linux Foundation `EasyCLA` tool](https://easycla.lfx.linuxfoundation.org/#/). Please note that some CCLAs require individuals/employees to be explicitly named on the CCLA.
```

Comment:
@robmoffat @scooter99boston shouldn't this be more specific and talk about the CSL CLA? (vs the generic FINOS ICLA).

Should we also link to the Contributor_License_Agreement.md file?

Reply:
HI Gab,  

I'd have to get @scooter99boston's opinion via email on this if you want it.  However, I'd like to point out that even though this is marked in green on the diff, this is exactly the same copy from the existing README.md file.  

Although it's not specific to the CSL CLA, I think it's correct in pointing out that contributors are going to have to tangle with EasyCLA, which seems to be the basic point.  

I'd like to leave it but your call.